VideoWeb/VideoWeb.AcceptanceTests/Features/HearingsList.feature

```diff
@@ -58,7 +58,7 @@ Scenario: Video Hearings Officer has 1 or more hearings
 	And a new browser is open for a Video Hearings Officer
 	When the user attempts to login with valid credentials
 	Then the user is on the VHO Venue List page
-  When the VHO selects the venue Birmingham Civil and Family Justice Centre
+  When the VHO selects the venue Automation Courtroom 01
```

Comment:
its no longer venues so that sentence doesn't look right? "the VHO selects the courtroom..."

Reply:
updated sentence